### PR TITLE
Setup Project

### PR DESCRIPTION
**Pull request by kevinchan159**

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+build/*
+node_modules/*
+docs/*
```
Comment (young-k) [inline .eslintignore]: remove `*` it's unnecessary

Comment (young-k) [inline .eslintignore]: alphabetize the folders

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+node_modules
+.vscode
+build
+.idea
+.env
```
Comment (young-k) [inline .gitignore]: alphabetize, add `/` for folders

File: src/routers/HelloWorldRouter.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { Request } from 'express';
+
+import ApplicationRouter from '../appdev/ApplicationRouter';
+
+class CatalogRouter extends ApplicationRouter<any> {
```
Comment (young-k) [inline src/routers/HelloWorldRouter.ts]: `CatalogRouter`?

File: env.template [hunk 4]
```diff
@@ -0,0 +1,8 @@
+export DB_USERNAME=FILL_IN
```
Comment (meganle) [inline env.template]: you don't need export

File: env.template [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+export DB_PASSWORD=FILL_IN
+export DB_HOST=localhost
+export DB_NAME=push-notifications
+export DB_PORT=5432
+export NODE_ENV=development
+export PORT=8000
+echo ‘dev env activated’
```
Comment (meganle) [inline env.template]: you can remove this

File: src/db/DBConnection.ts [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import { Connection, ConnectionOptions, createConnection } from 'typeorm';
+
+const isProduction = process.env.NODE_ENV === 'production';
+
+const entities = [
+];
```
Comment (meganle) [inline src/db/DBConnection.ts]: put this on one line

File: src/test/Main.test.js [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+    });
+});
+
+describe('AppDevUtilities', () => {
+    test('tryCheckAppDevURL', () => {
+        AppDevUtilities.tryCheckAppDevURL('/')
+    })
+});
+
+describe('ChronicleSession', () => {
```
Comment (meganle) [inline src/test/Main.test.js]: don't need chronicle or register

**Pull request merged**